### DISCUSSION TITLE
NHL LED Scoreboard d5e9 Network Security Key

### DISCUSSION THREAD

**Discussion by angtang10** [riffnshred/nhl-led-scoreboard#375] [General]
Could anyone help me figure out what password I use to connect to the scoreboard please?

**Comment by falkyre** [comment]
Are you using the image? If so, 12345678, it's in the readme.

**Comment by angtang10** [comment]
Yes I am. Okay awesome I'll try it when I get home from work. Many thanks!

Thanks,
Angelina
________________________________
